### PR TITLE
Splitter: add collapsedSize option on Central Pane on all demos

### PR DESCRIPTION
**Pull request by nikkithelegendarypokemonster**

### CODE REVIEW

File: apps/demos/Demos/Splitter/Overview/Vue/App.vue [hunk 1]
```diff
@@ -13,6 +13,7 @@
           :resizable="true"
           :collapsible="true"
           max-size="75%"
+          collapsedSize="8%"
```
Comment (Zedwag) [inline apps/demos/Demos/Splitter/Overview/Vue/App.vue]: collapsed-size

Comment (nikkithelegendarypokemonster) [reply apps/demos/Demos/Splitter/Overview/Vue/App.vue]: [8ef27a5](https://github.com/DevExpress/DevExtreme/pull/27316/commits/8ef27a5baac39668a21a903819e102b96d9172ba)

File: apps/demos/Demos/Splitter/Overview/React/App.tsx [hunk 2]
```diff
@@ -27,6 +27,7 @@ const App = () => (
             resizable={true}
             collapsible={true}
             maxSize="75%"
+            collapsedSize= "8%"
```
Comment (Zedwag) [inline apps/demos/Demos/Splitter/Overview/React/App.tsx]:
```suggestion
            collapsedSize="8%"
```

Comment (nikkithelegendarypokemonster) [reply apps/demos/Demos/Splitter/Overview/React/App.tsx]: [176a4aa](https://github.com/DevExpress/DevExtreme/pull/27316/commits/176a4aa209f9a1ddcb19dea993b796d86af091c8)

**Review by ksercs — DISMISSED** [review]

**Review by Zedwag — DISMISSED** [review]

**Review by Zedwag — APPROVED** [review]

**Review by ksercs — APPROVED** [review]

**Pull request merged**